item.py

```diff
@@ -6,7 +6,13 @@ def __init__(self, type_of_item, description):
         
 # give player points for using item
     def use_item(self, player):
+
+        #Checks the player's inventory for specific items and awards points accordingly.
+        
+        #If the player has a "tea_crate" item in their inventory, and uses it, they are awarded 5 points.
+        #If the player has a "strange bottle" item in their inventory, and uses it, they are awarded 10 points.
+
         if type_of_item == "tea_crate" and tea_crate in player.inventory:
```

Comment:
A couple issues here! First, you need to include the self key word to look at the type of item. So your if statement should be self.type_of_item. 

One way you could do this is:
`for item in player.inventory:
       if item.type_of_item == self.type_of_item:
          player.health += 5
`
What this will do is add 5 to the players health. Because currently, neither your item not your player has a points state. So you're not adding points to anything with what you have now. So you might want to add a points state to the player class if you want to update points not health. 

If you want to add different points based on the type of item, make points a state of item as well. Because then all you have to do is say `player.points += self.points`.

I'll make suggestions accordingly, but LMK if that isn't making sense!